src/sage/schemes/elliptic_curves/constructor.py

```diff
@@ -472,13 +472,25 @@ def create_object(self, version, key, **kwds):
             Keyword arguments are currently only passed to the
             constructor for elliptic curves over `\QQ`; elliptic
             curves over other fields do not support them.
+
+        EXAMPLES::
+            sage: E = EllipticCurve.create_object(0, (QQ, (1, 2, 0, 1, 2)), rank=2)
+            sage: type(E)
+            Elliptic Curve defined by y^2 + x*y = x^3 + 2*x^2 + x + 2 over Rational Field
+            sage: EllipticCurve.create_object(0, (GF(3), (1, 2, 0, 1, 2)), rank=2)
```

Comment:
Why repeat the same test in line 466?

The new test may better be in `TESTS::`.

Reply:
Indeed, probably a copy paste error somewhere.

The example is meant to address the note, but yes I suppose it doesn't need to be user-facing.

Reply:
I now saw the note. From my point of view, by the note, the error message for unexpected keyword arguments is not really necessary. But the note is not so visible to users. Hence I think the error message helps the users.